sorts/stalin_sort.py

```diff
@@ -0,0 +1,38 @@
+"""
+This is a pure Python implementation of the stalin sort algorithm.
+Stalin sort iterates over the list, comparing each element with the
+previous one, and removes any that are out of order.
+More information at: https://github.com/gustavo-depaula/stalin-sort
+"""
+
+
+def stalin_sort(collection: list[int]) -> list[int]:
+    """Pure python implementation of the stalin sort algorithm in Python
+    :param collection: a list of comparable items
+    :return: all of the items that were in order
+
+    Examples:
+    >>> stalin_sort([0, 5, 3, 2, 2])
+    [0, 5]
+    >>> stalin_sort([])
+    []
+    >>> stalin_sort([-2, -5, -45])
+    [-2]
+    """
+    if collection == []:
```

Comment:
All other falsy values are just as dangerous as an empty list.
```suggestion
    if not collection:
```